proposed/dotnet-bootstrapper-cli-experience.md

```diff
@@ -0,0 +1,256 @@
+3. The source/channel of each installation
+4. Dependencies linking installations to `global.json` files or direct DNVM commands
+
+### Install Terminology
+
+`Channel` is an overloaded term and previous products have been blocked from using the term `channel`.
+
+Channel is already used by Visual Studio.
+It is also used by the existing DNVM to describe an SDK as 'latest' or 'sts.'
+Utilizing the name `channel` may be tricky.
+
+In the broadest interpretation, a `channel` may end up meaning several things:
+- `Support Phase` and or `Release Type` (`lts` / `sts` / `preview`)
+- `Version` (`9.0`, `9.0.2xx`)
+- Something Else (`latest`)
+
+Other names:
+- Release
+- Ring
+
+### Competitor Offerings
+
+#### DNVM (https://github.com/dn-vm/dnvm/tree/main/src/dnvm)
+
+`dnvm install` Only supports a fully specified SDK version.
+`dnvm track <channel>` supports installing an SDK based on the channel.
+
+**Channels:**
+`lts`, `sts`, `preview`, `latest`.
+
+Installs created with a fully specified version are not update-able.
+Update terminology is not aware of the `global.json`.
+
+`dnvm restore` is the command that is aware of `global.json`, to the extent of installing based on the `rollForward` value and `sdk` of the file.
+`dnvm track` must be used to install utilizing a `channel`.
+
+`dnvm update` updates all tracked SDKS.
+
+#### NVM
+
+Not available on Windows, though `nvm-windows` is a product.
+`nvm use <version>` enables usage of a specific version of `node.js`.
+Installs at `NVM_DIR` which is `$HOME/.nvm`.
+`nvm install --lts`.
+
+https://github.com/nvm-sh/nvm
+
+#### RustUp
+`rustup install <toolchain>` installs based on `stable`, `nightly` or a fully specified version.
+`rustup update` updates based on `sts`.
+`rustup self update` updates itself.
+Modified the `PATH`.
+Installs are in `~/.cargo/bin`.
+
+More information on other internal docs.
+
+## ♻️ Update Experience
+
+### MVP
+`dotnet update` Updates all SDKS / Runtimes managed by `dnvm`. Uninstalls the older SDKs that are not installed as fully specified versions. Older SDKs are defined by ones that are no longer needed as specified by the summation of all known `global.json` files.
+
+For each possible update, `-y`, `-n` prompt is given.
+
+#### Options
+| Option | Description |
+|--------|-------------|
+| `-y --yes` | Auto accept |
+| `-w --what-if` | Show what would be updated, basically like auto 'no' |
+| `-h --help` | Show help |
+| `-u --url` | Use a custom `cdn` `releases.json` for locked down systems |
+
+#### Nice to Have
+| Command | Description |
+|---------|-------------|
+| `dotnet update version` | Only update the specified SDK managed by `dnvm` |
+| `dotnet self update` | Updates the `dnvm` product |
+
+#### Notes
+`dotnet track` is not provided.
+`dotnet untrack` is not provided.
+
+## 🗑️ Uninstall Experience
+
+`dotnet uninstall <channel>` - Uninstalls the specified SDK. Uninstall is simply a file deletion. No updates to the `PATH` are made.
```

Comment:
Perhaps revisit this part if this is the last remaining SDK / consider self uninstall of the toolchain.